src/extension.ts

```diff
@@ -1282,6 +1283,16 @@ async function createClusterKubernetes(request? : WizardUIRequest) {
     }
 }
 
+async function useContextKubernetes(explorerNode: explorer.ResourceNode) {
```

Comment:
As far as I know (and from what I found doing the first iteration of the tree), users can select commands from the palette even if the commands are targeted at the tree.  If that happens then `explorerNode` will be undefined.  It would be good to trap that situation and describe the intended usage (or display a pick list).

Reply:
See my answer for your first comment, actually the `Set to Current Cluster` command will not be visible at the dropdown command palette.